packages/template-tag-codemod/src/index.ts

```diff
@@ -97,22 +97,21 @@ export async function ensurePrebuild() {
 }
 
 export async function ensureAppSetup() {
-  let pkg: Package;
+  let filename = resolve(process.cwd(), 'package.json');
+  let content: string;
   try {
-    pkg = resolverLoader.resolver.packageCache.get(process.cwd());
+    content = readFileSync(filename, 'utf8');
   } catch (err) {
     console.error(`Run template-tag-codemod inside a Ember app.`);
     process.exit(-1);
   }
-  if (!pkg.packageJSON.exports) {
-    throw new Error(`must use package.json exports for self-resolvability. Plase add this to package.json:
-
- "exports": {
-    "./tests/*": "./tests/*",
-    "./*": "./app/*"
-  },
-
-`);
+  let json = JSON.parse(content);
+  if (!json.exports) {
+    json.exports = {
+      './tests/*': './tests/*',
+      './*': './app/*',
+    };
+    writeFileSync(filename, JSON.stringify(json, null, 2));
```

Comment:
this is touching files I would never expect a codemod to touch and messing with my indentation at the same time